tests/integration/src/test/java/org/apache/pulsar/tests/integration/containers/ElasticSearchContainer.java

```diff
@@ -23,24 +23,24 @@
 import java.util.Optional;
 
 public class ElasticSearchContainer extends ChaosContainer<ElasticSearchContainer> {
-    
+
     public static final String NAME = "ElasticSearch";
     static final Integer[] PORTS = { 9200, 9300 };
 
     public static final String IMAGE_NAME = Optional.ofNullable(System.getenv("ELASTICSEARCH_IMAGE"))
             .orElse("docker.elastic.co/elasticsearch/elasticsearch:7.15.0");
 
     public ElasticSearchContainer(String clusterName) {
-        super(clusterName, IMAGE_NAME);       
+        super(clusterName, IMAGE_NAME);
     }
-    
+
     @Override
     protected void configure() {
         super.configure();
         this.withNetworkAliases(NAME)
             .withExposedPorts(PORTS)
             .withEnv("discovery.type", "single-node")
-            .withEnv("ES_JAVA_OPTS", "-Xms2g -Xmx2g")
+            .withEnv("ES_JAVA_OPTS", "-Xms512m -Xmx1500m")
```

Comment:
Probably we should also remove the min heap size:

```suggestion
            .withEnv("ES_JAVA_OPTS", " -Xmx1500m")
```

Reply:
I guess it would be worth specifying some value for min heap size? I'll merge since this is flaky. We can revisit later if the problem persists.

Reply:
It's basically just an optimization to avoid JVM heap expansions when the process needs more memory. That's why for prod setting, we want to put `Xms == Xmx`. In the case of these tests, the ES process might not even need to get 500Mb of RAM, so we could save  on that by omitting the parameter altogether.